### ISSUE TITLE
Run in real time on Android device 

### ISSUE THREAD

**Issue by monschine** [xdspacelab/openvslam#240]
Hi, 

Is it possible to build an android app, and ./run_video_slam in real time ?
If it's possible, does the device would need a high specs to achieve that goal ?
Any hint on how to do that ?

**Comment by TheBricktop** [comment]
This is valid question, I would like to know that too.

**Comment by BijayanBhattarai** [comment]
i am waiting for answer

**Comment by Zumbalamambo** [comment]
+1

**Comment by williamliuok** [comment]
We had a really hard time to make it runnable on android, but we found it quite slow, We are still working on fugure out what's went worng

**Comment by TheBricktop** [comment]
@williamliuok  can You share Your work? Im very interested in making this work on more devices.

**Comment by williamliuok** [comment]
@TheBricktop Yeah,I can share my project on Github later, you guys might need it.

**Comment by Zumbalamambo** [comment]
+1

**Comment by iegorval** [comment]
@williamliuok In the end, did you have any luck running it on Android at some (near) real-time speed?

**Comment by jeronimoagullo** [comment]
@williamliuok Are there news about running it on Android?
